packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -230,59 +223,46 @@ export const ContentPanel: FC = () => {
-                                <SqlRunnerChart
-                                    data={queryResults}
-                                    config={barChartConfig}
-                                    isLoading={isLoading}
-                                    style={{
-                                        // NOTE: Ensures the chart is always full height
-                                        display:
-                                            selectedChartType ===
-                                            ChartKind.VERTICAL_BAR
-                                                ? 'block'
-                                                : 'none',
-                                        height: debouncedInputSectionHeight,
-                                        width: '100%',
-                                        flex: 1,
-                                    }}
-                                />
-                            )}
-
-                            {queryResults?.results && pieChartConfig && (
-                                <SqlRunnerChart
-                                    data={queryResults}
-                                    config={pieChartConfig}
-                                    isLoading={isLoading}
-                                    style={{
-                                        // NOTE: Ensures the chart is always full height
-                                        display:
-                                            selectedChartType === ChartKind.PIE
-                                                ? 'block'
-                                                : 'none',
-                                        height: debouncedInputSectionHeight,
-                                        width: '100%',
-                                        flex: 1,
-                                    }}
-                                />
-                            )}
-
-                            {queryResults?.results && (
-                                <Paper
-                                    shadow="none"
-                                    radius={0}
-                                    p="sm"
-                                    sx={() => ({
-                                        flex: 1,
-                                        overflow: 'auto',
-                                    })}
-                                >
-                                    <Table
-                                        data={queryResults.results}
-                                        config={tableVisConfig}
+                            {queryResults?.results &&
+                                currentVisConfig &&
+                                // TODO: need a more robust check for what can be rendered
+                                // by the chart renderer. For now its just not tables
+                                !isTableChartSQLConfig(currentVisConfig) && (
```

Comment:
a different type guard that encompasses the 3 types we have at the moment? `LINE` `BAR` and `PIE`?

Reply:
Yup, incoming!